iot/pom.xml

```diff
@@ -27,8 +27,11 @@
     <module>iot-application-base</module>
     <module>iot-auth-service</module>
     <module>iot-device-registry-file</module>
+    <module>iot-amqp-adapter</module>
```

Comment:
Do we need AMQP adapter as part of this pr?

Reply:
Nope. Should be gone now.